quickwit/quickwit-cli/src/index.rs

```diff
@@ -323,11 +334,18 @@ impl IndexCliCommand {
         } else {
             None
         };
+        let commit_type = match (matches.is_present("wait"), matches.is_present("force")) {
+            (false, false) => CommitType::Auto,
+            (false, true) => CommitType::Force,
+            (true, false) => CommitType::WaitFor,
+            (true, true) => bail!("Can't specify both --wait and --force"),
```

Comment:
You can probably bail out even earlier during command line parsing by using ArgGroup or the conflicts_with method.